packages/init/src/SliceMachineInitProcess.ts

```diff
@@ -215,6 +223,31 @@ export class SliceMachineInitProcess {
 		}
 	}
 
+	protected async checkIsLoginRequired(): Promise<boolean> {
+		try {
+			assertExists(this.context.repository, "");
```

Comment:
With what I saw you just need to put again the same message as the other places:

```suggestion
			assertExists(
			   this.context.repository,
			   "Repository selection must be available through context to proceed",
		          );
```

Reply:
It's not necessary here because the assertion is caught and is used to return a need for login!

Reply:
I don't understand? For me it's just that all places I saw it was repeating the text even if not needed, no?

Reply:
Oh I understand!